install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js

```diff
@@ -558,6 +615,21 @@ const ExtensionsComponent = () => {
             </div>
           )}
         </SectionWrapper>
+
+
+        {/* 
+        
+        // Feedback component is comment currently because the api required to use this authentication error
```

Comment:
Currently i have commented out the feedback component though the functionality is there until we able to use the api. II tried to set the cookie in header but it didn't solved. // @MUzairS15 @humblenginr